strategy/json/sql/postgres/single_stream_strategy.go

```diff
@@ -36,39 +38,35 @@ func NewSingleStreamStrategy(converter goengine.MessagePayloadConverter) (sql.Pe
 func (s *SingleStreamStrategy) CreateSchema(tableName string) []string {
 	tableName = postgres.QuoteIdentifier(tableName)
 
-	statements := make([]string, 3)
+	statements := make([]string, 2)
 	statements[0] = fmt.Sprintf(
 		`CREATE TABLE %s (
     no BIGSERIAL,
     event_id UUID NOT NULL,
     event_name VARCHAR(100) NOT NULL,
     payload JSON NOT NULL,
-    metadata JSONB NOT NULL,
+    metadata JSONB NOT NULL, 
+    aggregate_type VARCHAR(50) NOT NULL,
+	aggregate_id UUID NOT NULL,
+	aggregate_version SMALLINT NOT NULL,
     created_at TIMESTAMP(6) NOT NULL,
     PRIMARY KEY (no),
-    CONSTRAINT aggregate_version_not_null CHECK ((metadata->>'_aggregate_version') IS NOT NULL),
-    CONSTRAINT aggregate_type_not_null CHECK ((metadata->>'_aggregate_type') IS NOT NULL),
-    CONSTRAINT aggregate_id_not_null CHECK ((metadata->>'_aggregate_id') IS NOT NULL),
     UNIQUE (event_id)
 );`,
 		tableName,
 	)
-	statements[1] = fmt.Sprintf(
-		`CREATE UNIQUE INDEX ON %s
-((metadata->>'_aggregate_type'), (metadata->>'_aggregate_id'), (metadata->>'_aggregate_version'));`,
```

Comment:
Why was this index removed?

Reply:
Should we keep this index on metadata? As far as we know, we'll be using this column for different purpose in future (not just for saving `aggregate_id`, `aggregate_type` and `aggregate_version`), or did we misunderstood?

Reply:
No but we should keep and index on type, id and version

Reply:
https://github.com/hellofresh/goengine/pull/50/commits/84533923e2107df9f94cb26ccffd94b87a521a9f